ddl/db_integration_test.go

```diff
@@ -3162,6 +3162,165 @@ func (s *testIntegrationSuite3) TestDropTemporaryTable(c *C) {
 	tk.MustQuery("select * from a_local_temp_table_8").Check(testkit.Rows())
 }
 
+func (s *testIntegrationSuite3) TestDropGlobalTemporaryTableWithDropGlobalStmt(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+	tk.MustExec("use test")
+	tk.MustExec("set tidb_enable_noop_functions=true")
+	tk.MustExec("set tidb_enable_global_temporary_table=true")
+	clearSQL := "drop table if exists tb, tb2, temp, temp1, ltemp1, ltemp2"
+	tk.MustExec(clearSQL)
+	defer tk.MustExec(clearSQL)
+	// two normal table test.tb, test.tb2, a temporary table with name test.tb2
+	tk.MustExec("create table tb(id int)")
+	tk.MustExec("create table tb2(id int)")
+	// two global temporary table test.temp, test.temp1
+	tk.MustExec("create global temporary table temp(id int) on commit delete rows")
+	tk.MustExec("create global temporary table temp1(id int) on commit delete rows")
+	// two local temporary table test.ltemp1, test.ltemp2
+	tk.MustExec("create temporary table ltemp1(id int)")
+	tk.MustExec("create temporary table ltemp2(id int)")
```

Comment:
In function `TestDropGlobalTemporaryTableWithDropGlobalStmt`, local temporary table is ... covered.

Reply:
This is useful to test dropping a local temporary table with statement `drop global temporary table...`